api/src/test/java/org/openmrs/api/OrderServiceTest.java

```diff
@@ -3790,6 +3800,29 @@ public void saveOrderGroupAttributeType_shouldSaveOrderGroupAttributeTypeGivenOr
 		assertNotNull(orderGroupAttributeType.getId());
 		assertEquals(initialGroupOrderAttributeTypeCount+1,Context.getOrderService().getOrderGroupAttributeTypes().size());
 			}
+			
+			@Test
+			public void saveOrderGroupAttributeType_shouldEditAnExistingOrderGroupAttributeType(){
+				executeDataSet(ORDER_GROUP_ATTRIBUTES);
+				final String UUID4 ="9cf1bdb2-d18e-11ea-87d0-0242ac130003";
+				final String name ="ECG";
+				//Check for values in the database
+				OrderGroupAttributeType orderGroupAttributeType=Context.getOrderService().getOrderGroupAttributeTypeById(4);
+				assertEquals(UUID4,orderGroupAttributeType.getUuid());
+				assertEquals(name,orderGroupAttributeType.getName());
+				assertEquals("Testing unretire",orderGroupAttributeType.getRetireReason());
+                //edit existing values in the database
+				orderGroupAttributeType.setRetireReason("Change Order To Laparascopy");
+				orderGroupAttributeType.setName("Laparascopy");
+				orderService.saveOrderGroupAttributeType(orderGroupAttributeType);
+                //confirm new values are persisted
```

Comment:
do you need these commented out lines

Reply:
FYI i was cleaning out wildcards , if closely investigate the link i shared , order was not a priority though , honestly speaking it makes the imports look neat. 

That's why you have probably seen some changes .About the documentation , i didn't mean the Javadocs rather the openmrs wiki.

If you thinks the inline comments are unnecessary you can suggest that on the Dev forum so we clean them out and, i personally think they ease the reviewiers task for long bodied methods .

Just thinking out loud.